tools/distpartitioning/convert_partition.py

```diff
@@ -18,6 +18,36 @@ def create_dgl_object(graph_name, num_parts, \
     This function creates dgl objects for a given graph partition, as in function
     arguments. 
 
+    The input schema object is structure as follows: 
+    1. "nid" is a dictionary, which has the following structure. 
```

Comment:
The key value pairs in the "nid" dict has the following format.

Reply:
modified the text accordingly.